src/webportal/src/app/job/job-view/job-view.component.js

```diff
@@ -490,6 +492,23 @@ $(document).ready(() => {
     loadJobs(query['vcName']);
     $('#content-wrapper').css({'overflow': 'hidden'});
   }
+  $(document).on('click', '#resubmitjob_btn', () => {
+    let url;
+    let configYaml = yaml.safeLoad(configInfo);
+    let username = $('#nav-username').text();
```

Comment:
Use `cookies.get("user")` to get the current user without introduce other element ids